react/finger-exercises/redux/01 - actions and reducers/books/actions.js

```diff
@@ -9,7 +9,8 @@ related to @@BOOKS for easier debugging.
 */
 export const actions = {
   SET_FAVORITE_BOOK: '@@BOOKS/SET_FAVORITE_BOOK',
-  ADD_READ_BOOK: '@@BOOKS/ADD_READ_BOOK'
+  ADD_READ_BOOK: '@@BOOKS/ADD_READ_BOOK',
+  SET_MOST_HATED_BOOK: '@@BOOKS/MOST_HATED_BOOK'
```

Comment:
For more consist I think that is better the action rename to `'@@BOOKS/SET_MOST_HATED_BOOK'`